buying_selling/urls.py

```diff
@@ -1,15 +1,17 @@
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 from django.conf import settings
 from django.conf.urls.static import static
 from users import views as user_views
 from rest_framework_simplejwt.views import TokenRefreshView
 
 urlpatterns = [
-    path("admin/", admin.site.urls),
+    path('api-auth/', include('rest_framework.urls')),
```

Comment:
This line is probably not required. Verify that once

Reply:
Yes it isn't required, I'll remove it